packaging/docker/docker-compose.yml

```diff
@@ -1,21 +1,26 @@
 version: "3.4"
 
 services:
+  baseimage:
```

Comment:
What's the reason for having this included here? As it is a service, it should probably have a different name, if we want to keep it here.